src/User.php

```diff
@@ -346,4 +348,18 @@ public function can(string $permissionName, array $params = []): bool
 
         return $this->accessChecker->userHasPermission($this->getId(), $permissionName, $params);
     }
+
+    public function authTimeout(int $timeout = null): self
```

Comment:
Is it setter? Why not `setAuthTimeout()`?

Reply:
Also for method `absoluteAuthTimeout`.